docs/admin/backup.rst

```diff
@@ -19,9 +19,9 @@ the cloud. The backups can be controlled in the management interface on the
 
 The backups using Borg are incremental and Weblate is configured to keep following backups:
 
-* 14 daily backups
-* 8 weekly backups
-* 6 monthly backups
+* 14 day backups
```

Comment:
Assuming every 14 days, (not 14 times in one day).
Not quite sure this is better.

Reply:
Yes - backup is executed daily, it keeps 1 backup for each of last 14 days, 8 weeks, ...

Reply:
IMHO this makes it more confusing...
* *…is configured to keep `14 daily backups`* == it keeps 14 backups which are made every day; older one are getting removed
* *…is configured to keep `14 day backups`* == ???

Reply:
Yes, I reworded it to stay clarified.